### ISSUE TITLE
CMake CMP0022 "OLD" deprecation warning

### ISSUE THREAD

**Issue by moonheart08** [DFHack/dfhack#1182]
```➜  build git:(twbt_experiments) ✗ cmake .. -DCMAKE_BUILD_TYPE:string=Release -DCMAKE_INSTALL_PREFIX=../df_linux
CMake Deprecation Warning at CMakeLists.txt:5 (CMAKE_POLICY):
  The OLD behavior for policy CMP0022 will be removed from a future version
  of CMake.

  The cmake-policies(7) manual explains that the OLD behaviors of all
  policies are deprecated and that a policy should be set to OLD only under
  specific short-term circumstances.  Projects should be ported to the NEW
  behavior and not rely on setting a policy to OLD.


CMake Error at CMakeLists.txt:134 (message):
  In-source builds are not allowed.


-- Configuring incomplete, errors occurred!
See also "/home/moony/Projects/dfhack/CMakeFiles/CMakeOutput.log".      
```

CMake version: 3.9.4
Linux Distro: Arch (Updated 5 minutes ago)
DF Version: Latest

**Comment by lethosor** [comment]
You need to build in a separate folder (see Compile.rst).

**Comment by moonheart08** [comment]
Hurr. Im stupid. Thanks.

**Comment by lethosor** [comment]
(Repurposing this issue - the first warning isn't an error yet, but will be at some point, so we still need to address that.)

**Comment by lethosor** [comment]
Fixed in #1233
